t.stream/balancer.t

```diff
@@ -0,0 +1,220 @@
+# vim:set ft= ts=4 sw=4 et fdm=marker:
```

Comment:
I don't like the `t.stream` directory. Can we use `t/stream/` instead?

Reply:
We can certainly do that. Only concern is that since the `stream-lua-nginx-module` containing the balancer implementation has not been released yet, putting it under `t/` will cause Travis tests to turn red. I think we can change this later but merge first?

Reply:
@dndx I'll take care of the merging thing in stream-lua-nginx-module.